setup/jetstream/streams.go

```diff
@@ -89,4 +91,9 @@ var streams = []*nats.StreamConfig{
 		Retention: nats.InterestPolicy,
 		Storage:   nats.FileStorage,
 	},
+	{
+		Name:      OutputPresenceEvent,
+		Retention: nats.InterestPolicy,
+		Storage:   nats.FileStorage,
```

Comment:
We probably want `MemoryStorage` for this particular stream, since I don't imagine we care about sending delayed presence updates if Dendrite were to die. We'd just end up generating new ones.

Reply:
We could also potentially set a `MaxAge`, like `OutputTypingEvent`, so that if an update becomes stale after a certain period of time, we just drop it.

Reply:
Sounds like a good idea.
We probably also need to have a look at the FederationAPI later, as presence events could pile up for servers and we'd send them old events.

Reply:
The question is, what is a good `MaxAge` for the stream?

Reply:
I suppose however long we consider a presence update to be "stale". Is that in the order of minutes or so? Does the spec say how often we should be sending things?

Reply:
The spec just says 
> The server will automatically set a user’s presence to unavailable if their last active time was over a threshold value (e.g. 5 minutes).

that's the 5min cleanup period which changes the presence.